src/Views/batchprocessor.html

```diff
@@ -5,7 +5,12 @@
     <button class="close" ng-click="vm.Close()">
         &times;
     </button>
-    <h4 class="modal-title">StreamStats Batch Processor: Preview Version</h4>
+    <h4 class="modal-title">StreamStats Batch Processor  
+        <i ng-click = "showTooltip=!showTooltip" class="fa fa-question-circle"></i>

```

Comment:
I would add a little space between "StreamStats Batch Processor" and the icon 
![image](https://github.com/USGS-WiM/StreamStats/assets/40237491/2f6fc231-c850-43f7-adae-4f128be0048a)